docs/developers/fuse-box/flutter-sdk/4-send-batch-txn.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Send Batch Transactions
+sidebar_position: 4
+---
+
+### Non-Blocking Transactions
+
+A batch refers to a group of calls inside a single user operationhe. This can be an approve and send transaction in one call. The executeBatch() method is used.
+
+The executeBatch() method takes in an array of _`Call()`_. With each method taking in 3 keys: to, value and data.
+
+```javascript
+executeBatch([
```

Comment:
@emmaodia That's incorrect. In Dart it should be:

```dart
.executeBatch([
    Call(to: to, value: value, data: data),
    Call(to: to, value: value, data: data),
    Call(to: to, value: value, data: data),
  ]);
```

Reply:
Fixed.